src/js/server/apply/router.js

```
@@ -87,13 +88,39 @@ applyRouter.post('/team', fileUploadMiddleware.none(), (req, res, next) => {
   });
 });
 
+// Process the RSVP response
+applyRouter.post('/rsvp', auth.requireAuth, function(req, res) {
+  const rsvp = req.body.rsvp;
+  if (rsvp) {
+    req.user.getHackerApplication().then(hackerApplication => {
+      if (hackerApplication == null) {
+        return Promise.resolve(null);
+      } else {
+        return hackerApplication.getApplicationResponse();
+      }
+    }).then(applicationResponse => {
+      if (applicationResponse != null) {
+        return rsvpToResponse(applicationResponse, rsvp);
+      } else {
+        return Promise.resolve(null);
+      }
+    }).then(() => {
+      res.redirect('/apply//dashboard');
+    }).catch(() => {
+      res.redirect('/apply/dashboard');
```

Do we want to silently swallow all errors here?